sympy/core/tests/test_args.py

```diff
@@ -2363,6 +2363,9 @@ def test_sympy__functions__special__beta_functions__beta():
     from sympy.functions.special.beta_functions import beta
     assert _test_args(beta(x, x))
 
+def test_sympy__functions__special__beta_functions__betainc():
+    from sympy.functions.special.beta_functions import betainc
+    assert _test_args(betainc(x, x, x))
 
```

Comment:
There should be additional blank lines before and after this to match the spacing in the rest of the module.